charts/workspacemanager/templates/proxy-configmap.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: terra-workspace-manager-proxy-configmap
+  labels:
+    {{ if not .Values.devDeploy }}{{ template "workspacemanager.labels" }}{{ end }}
+data:
+  siteconf: |-
```

Comment:
Related somewhat to the comment earlier about site.conf being the filename, would it be beneficial for other people who encounter this file if it had the following name: `terra-workspace-manager-proxy-apache-httpd-configmap`?

Reply:
Renamed to `proxy-apache-httpd-configmap`. Adding workspace manager feels redundant given the directory structure.